### DISCUSSION TITLE
Discord Server for Geany

### DISCUSSION THREAD

**Discussion by zhaolinlau** [geany/geany#3285] [General]
I think Geany should have a discord server instead of matrix server because most people & open source organizations like vue, laravel, electron, they use discord as their community server & conversation platform. And now Geany's matrix server looks dying bruh, so I prefer to have a discord server instead. It would help Geany having a larger community in the future.

**Comment by elextr** [comment]
AFAIK the matrix is only used for the interactions used for making releases, I don't think it is inhabited by devs outside that period.

**Reply by zhaolinlau** [reply]
Yes I know that's why I prefer discord. It could help to grow the community of Geany because you can make announcements/update news like new releases and bug fixes.

**Reply by eht16** [reply]
> AFAIK the matrix is only used for the interactions used for making releases, I don't think it is inhabited by devs outside that period.

Not exclusively, some devs guys are hanging around and also a few dozens of regular users as well as casual users to ask their questions.
I didn't make a list but I think some of the previous regular IRC users are now also in the Matrix channel.

> I think Geany should have a discord server instead of matrix server because most people & open source organizations like vue, laravel, electron, they use discord as their community server & 

The listed projects probably have a quite disjunct community from Geany, I'd say. And as said above, the Matrix channel is already used by some people, so there is no real need to spend time and efforts in changing the chat software again(!).

Most importantly, unlike Matrix is Discord proprietary software and at least for me this would be a full no-go.
